pycaret/containers/models/time_series.py

```diff
@@ -203,18 +204,22 @@ def __init__(self, globals_dict: dict) -> None:
         np.random.seed(globals_dict["seed"])
         gpu_imported = False
 
-        from sktime.forecasting.naive import NaiveForecaster
+        from sktime.forecasting.naive import NaiveForecaster  # type: ignore
 
         args = {}
         tune_args = {}
+        sp = globals_dict.get("seasonal_parameter")
+        sp = sp if sp is not None else 1
```

Comment:
perhaps this can be summarized in one line 

```python
sp = global_dict.get("seasonal_parameter", 1)
```

Reply:
I tried that. I think the default is set to None and it just returns None in that case. So I had to add the extra line.

Reply:
![image](https://user-images.githubusercontent.com/33585645/115120859-4a366180-9f75-11eb-9aa8-65faa1c1b436.png)

Reply:
yes, makes sense